gulpfile.js

```diff
@@ -30,6 +30,12 @@ gulp.task('copy-vendors', function () {
 		.pipe(gulp.dest('./public/js'));
 });
 
+gulp.task('copy-assets', function () {
```

Comment:
Unless you want to make separate use of them later, say when your repo gets
larger?
On 20 Mar 2015 02:22, "Pavel" <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/themacclesoft/ractive-foundation/pull/34#discussion_r26764095>
> :
>
> > @@ -30,6 +30,12 @@ gulp.task('copy-vendors', function () {
> >  		.pipe(gulp.dest('./public/js'));
> >  });
> >
> > +gulp.task('copy-assets', function () {
>
> Could we make use of copy vendors task ? Merge them together perhaps.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/themacclesoft/ractive-foundation/pull/34/files#r26764095>
> .
>